src/main/java/domain/board/File.java

```diff
@@ -0,0 +1,54 @@
+package domain.board;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public final class File {
+
+    private static final Map<Integer, File> CACHE = new HashMap<>();
+    private final int value;
```

Comment:
```suggestion
    private static final Map<Integer, File> CACHE = new HashMap<>();
    
    private final int value;
```
캐시를 적용해주셨군요! 도입하신 이유가 궁금합니다.

Reply:
File과 Rank의 개수가 8개로 고정되어 있기 때문에 매번 새로운 객체를 생성하는 것보다 캐시를 사용하는 것이 효율적이라고 생각해서 도입하였습니다!